src/Commands/Load.php

```diff
@@ -33,7 +33,7 @@ public function handle()
         $useLatestSnapshot = $this->option('latest') ?: false;
 
         $name = $useLatestSnapshot
-            ? $snapShots->last()->name
+            ? $snapShots->first()->name
```

Comment:
The list of snapshots is sorted descending. The latest created snapshot will be the first item in the list.